code/_globalvars/lists/mobs.dm

```diff
@@ -12,3 +12,5 @@ var/global/list/mob_list = list()					//all mobs, including clientless
 var/global/list/living_mob_list = list()			//all alive mobs, including clientless. Excludes /mob/new_player
 var/global/list/dead_mob_list = list()				//all dead mobs, including clientless. Excludes /mob/new_player
 var/global/list/joined_player_list = list()			//all clients that have joined the game at round-start or as a latejoin.
+
+var/global/list/all_antagonists = list()			//all minds of all antagonists, past and present
```

Comment:
Yeah there's some cases of "I forgot I made this var already and made it again" that I'll rectify before this gets anywhere close to merging